### ISSUE TITLE
"Cannot read property 'split' of undefined' Version 59

### ISSUE THREAD

**Issue by C-hoang** [six7/figma-tokens#208]
Hi there,

I was in the process of creating tokens of all colours and related alias for components in the design system I've been working on. It was an extensive list of tokens and I was in the middle of creating a colour token that was going to reference another. When I hit enter, I was greeted with this screen:

![image](https://user-images.githubusercontent.com/89397474/130454511-9141e7e0-b049-402e-8873-ed2b5763f3fd.png)

The only setting I would have changed was switching it from 'Apply to page' to 'Apply to selection' as it meant I could create tokens faster without every new token trying to apply it to the whole page. 

Now every time I try and open the plugin now it shows the welcome screen for a split second before throwing up the same error message as above.

I've opened and closed it, tried shutting Figma down for an extended period of time and opening it up again with the plugin, same problem. Seems to be a file specific problem as I can launch the plugin no problems on other files.

I haven't tried uninstalling and reinstalling the plugin yet as I'm afraid of losing all my work if I do.

Unfortunately, I can't link you the Figma file

Thanks for any help you can provide.

**Comment by six7** [comment]
When you open the plugin can you press F12 (open developer console) and paste the content here? There should be a "Here's the tokens..." followed by a JSON. We can then try to debug the cause of this, also this is sort of your "backup" for when things go wrong - you can copy that JSON and find the error and fix it. But it'd help me to debug this for everyone.

**Comment by C-hoang** [comment]
> When you open the plugin can you press F12 (open developer console) and paste the content here? There should be a "Here's the tokens..." followed by a JSON. We can then try to debug the cause of this, also this is sort of your "backup" for when things go wrong - you can copy that JSON and find the error and fix it. But it'd help me to debug this for everyone.

First time dealing with this part of Figma, hope this is what you're after

    VM148:120 TypeError: Cannot read property 'split' of undefined
    at yd (<anonymous>:120:379630)
    at ta (<anonymous>:79:58816)
    at Wu (<anonymous>:79:111288)
    at Sc (<anonymous>:79:98264)
    at wc (<anonymous>:79:98192)
    at bc (<anonymous>:79:98055)
    at fc (<anonymous>:79:95042)
    at <anonymous>:79:44711
    at t.unstable_runWithPriority (<anonymous>:87:3768)
    at Zo (<anonymous>:79:44488)
    at qo (<anonymous>:79:44656)
    at Jo (<anonymous>:79:44591)
    at dc (<anonymous>:79:95363)
    at Object.notify (<anonymous>:120:77038)
    at e.t.notifyNestedSubs (<anonymous>:120:77674)
    at e.t.handleChangeWrapper (<anonymous>:120:77742)
    at m (<anonymous>:37:14583)
    at Object.dispatch (<anonymous>:120:439107)
    at Object.setActiveTab (<anonymous>:120:438593)
    at <anonymous>:120:427682
    at s (<anonymous>:51:147184)
    at Generator._invoke (<anonymous>:51:146937)
    at Generator.next (<anonymous>:51:147543)
    at gv (<anonymous>:120:426381)
    at a (<anonymous>:120:428807)
    at <anonymous>:120:428868
    at new Promise (<anonymous>)
    at <anonymous>:120:428747
    at <anonymous>:120:428911

There is also this below it 
![image](https://user-images.githubusercontent.com/89397474/130616294-0f2f4a7c-ed45-4752-959d-c0c09052d18e.png)

**Comment by six7** [comment]
And there's nothing like `Received tokens` followed by your token JSON?

Could you duplicate the file, remove every page and every node and then share it with me? That way I'd have access to your tokens without you sharing sensible information with me.

**Comment by C-hoang** [comment]
Yep, not a problem. I've created a duplicate and removed almost everything and invited you
